.github/workflows/release.yaml

```diff
@@ -1,10 +1,5 @@
 name: Release
 
-on:
```

Comment:
Can you add a manual trigger for now? See https://github.com/Uniswap/interface/blob/main/.github/workflows/release.yaml#L5-L6